src/content/platform-integration/android/install-android/install-android-from-ios.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Add Android devtools for Flutter from iOS start
+title: Add Android as a target platform for Flutter from iOS start
```

Comment:
what does "from iOS start" mean? Do you mean "from iOS" or "on iOS"?

Reply:
These pages would most likely be found from their original install page. 

For example, a new user goes to Get Started > Choose OS > macOS choose target > iOS. This is "iOS start".

There's a link at the bottom for them to add Android to their build as a next step, which is why this page says "from iOS start". 

It _could_ say "with iOS target tools already installed", but that seemed like a mouthful!

Reply:
I still don't like "from iOS start". Why can't it say "on iOS"? Are there multiple "on iOS" pages?